environments/common/src/main/java/org/jboss/weld/environment/deployment/discovery/ClassPathBeanArchiveScanner.java

```diff
@@ -93,6 +93,9 @@ public ClassPathBeanArchiveScanner(Bootstrap bootstrap) {
                 continue;
             }
             File entryFile = new File(entry);
+            if (!entryFile.exists()) {
```

Comment:
I'd prefer to log a warning if a file does not exist...

Reply:
Oh, yeah, forgot about it, sorry.
Added now.